.vscode/settings.json

```diff
@@ -17,9 +17,7 @@
   "[jsonc]": {
     "editor.defaultFormatter": "esbenp.prettier-vscode"
   },
-  "editor.codeActionsOnSave": {
-    "source.fixAll": true
-  },
+  "editor.codeActionsOnSave": { "source.fixAll": true },
```

Comment:
I wonder if auto save fires during i. e. typescript build when new js files are created? Unless dist folder is ignored, I'm not sure

Reply:
This setting works only in the editor so wouldn't be execute during ts build

Reply:
Nvm, it's vscode specific